### ISSUE TITLE
Error with parameterized `Int!` argument using reason-apollo-client

### ISSUE THREAD

**Issue by JasoonS** [reasonml-community/graphql-ppx#177]
I'm not 100% sure this is an issue with the graphql-ppx, there is some chance it is an issue with reason-apollo-client.

I'll just give you the steps to reproduce the error. I won't try explain it more than that, because I'm pretty clueless what it could be. (I have hosted the gql backend on my own server for convenience - http://34.65.171.221:8000/subgraphs/name/wildcards/integration-spike/graphql - ping me if it doesn't work for any reason)

```
git clone -b js/gql-ppx-issue git@github.com:wildcards-world/graph-integration-test-spike.git
cd graphql-connection-utils
yarn
yarn re:build
yarn test # This passes!
yarn test2 # This fails!
```

You'll notice that `yarn test` runs `mocha src/Test.bs.js` and `yarn test2` runs `mocha src/Test2.bs.js`.

This is the diff between those two files, Test.re on the left Test2.re on the right (notice they are identical, but in `Test2.re` I have commented out some code). Everything compiles.

![image](https://user-images.githubusercontent.com/6032276/88159457-9d110180-cc0d-11ea-8840-fd2ac6a7d574.png)

![image](https://user-images.githubusercontent.com/6032276/88159526-b44fef00-cc0d-11ea-8dfa-b4b7f1feba40.png)

![image](https://user-images.githubusercontent.com/6032276/88159591-c6319200-cc0d-11ea-8f14-6c5a95500558.png)

Tldr: the code in green on the right causes a runtime failure.

Thanks for all the work you've put into this PPX! Massive respect to you :)

**Comment by JasoonS** [comment]
False alarm - it is an error with thegraph.com . Thank for your patience

**Comment by JasoonS** [comment]
https://github.com/graphprotocol/graph-node/issues/1460 - this is the issue
